### ISSUE TITLE
Serve all the vendor assets after bundling them into one single combined.min.js file

### ISSUE THREAD

**Issue by icyflame** [metakgp/mfqp-source#3]
- Replace all the CDN's includes with this one file.

**Comment by icyflame** [comment]
@athityakumar Do you think you can work on this? It's an extremely small issue, but will do loads for optimising this page.

**Comment by athityakumar** [comment]
Are you talking about minifying the js scripts just once (by running some
ruby script locally)?

**Comment by icyflame** [comment]
@athityakumar No, this repo is working using `grunt gh-pages`, which is a task defined in the `Gruntfile.js`. You should have a look at that. 

I am saying that we should add a grunt task that will minify everything and then, running `grunt && git push origin master && grunt gh-pages` should deploy the minified version of the assets on this repository to the `gh-pages` branch of this repo. (have a look at the `gh-pages` branch for more details on this, and also the `grunt gh-pages` plugin homepage)
